tests/Doctrine/Tests/DBAL/Functional/Driver/SQLSrv/StatementTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Functional\Driver\SQLSrv;
+
+use Doctrine\DBAL\Driver\SQLSrv\Driver;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+class StatementTest extends DbalFunctionalTestCase
+{
+    protected function setUp()
+    {
+        if (!extension_loaded('sqlsrv')) {
+            $this->markTestSkipped('sqlsrv is not installed.');
+        }
+
+        parent::setUp();
+
+        if (!$this->_conn->getDriver() instanceof Driver) {
+            $this->markTestSkipped('sqlsrv only test');
+        }
+    }
+
+    public function testFailureToPrepareResultsInException()
+    {
+        // use the driver connection directly to avoid having exception wrapped
+        $stmt = $this->_conn->getWrappedConnection()->prepare(null);
+
+        // it's impossible to prepare the statement without bound variables for SQL Server,
+        // so the preparation happens before the first execution when variables are already in place
+        $this->setExpectedException('Doctrine\\DBAL\\Driver\\SQLSrv\\SQLSrvException');
```

Comment:
You can use `SQLSrvException::class` here, since I think that we bumped version requirements to minimum PHP 5.5